package-lock.json

```diff
@@ -3244,18 +3228,15 @@
         },
         "code-point-at": {
           "version": "1.1.0",
-          "bundled": true,
-          "optional": true
+          "bundled": true
         },
         "concat-map": {
           "version": "0.0.1",
-          "bundled": true,
-          "optional": true
```

Comment:
any idea why these changes keep bouncing around? different versions of `npm`?

Reply:
@abeisgoat explain JavaScript to me plz

Reply:
tldr: Please use npm >= 6.9.0. All reviewers should ask for npm >= 6.9.0 if they see diffs like this. This is documented at https://github.com/firebase/firebase-tools/blob/master/.github/CONTRIBUTING.md#development-setup

Reply:
Thanks Yuchen!